.npmignore

```diff
@@ -1,5 +1 @@
-*
-!lib/*
-!package.json
-!tsconfig.json
-!README.md
\ No newline at end of file
+
```

Comment:
This change is for what?

Reply:
But I think that we don't want to publish all files to npm. Only `lib` folder, `package.json`, `README.md`

Reply:
That's different now, because it will be published as an Angular module. Before publishing, the module will be generated in dist and we should publish everything in there.